### ISSUE TITLE
Do you have plans to open source the building height calculation algorithm?

### ISSUE THREAD

**Issue by jharpster** [opencitymodel/opencitymodel#10]
Do you have plans to open source the building height calculation algorithm?

**Comment by agilliland** [comment]
Yes, absolutely!  We have a repository which has all of our existing code for executing the pipeline already available https://github.com/opencitymodel/data-pipeline

Currently the step in the pipeline where we apply the height modeling isn't in that repo because we've been changing that code very regularly and it's currently a Spark job being run on a Databricks notebook 😱.  Once we get a little further along and things start to solidify then I'll pull that code out of the notebook and put it into the repo.

I'd expect that to happen in the next 2-3 months for sure.

**Comment by jharpster** [comment]
@agilliland thanks for the update! Great work, our team was looking at it today.
